content/blog/2023/08/28/2023-08-28-renovate-bote-probe-blog.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Renovate Probe
+tags:
+- gsoc
+- gsoc2023
+- healthscore
+- probes
+- plugin
+authors:
+- jagruti
+opengraph:
+  image: /images/gsoc/2023/jagruti/renovate-probe.jpg
+sig:
+- gsoc
+---
+
+== Introduction
+
+Bots are used to automate tasks in a plugin. Plugin Health Scoring Systems tracks usage of two bots - Dependabot, link:https://docs.renovatebot.com/[Renovate].
+
+Renovate probe checks whether Renovate is configured in a plugin.
+
+== Importance of the probe
+
+This a community requested probe to track the usage of Renovate.
+
+The pull requests created by Renovate will be counted when scoring the probe.
+
+== Challenges
+
+This probe required creating an `abstract` class and reconfiguring the Dependabot class to use the new `abstract` class.
+
+Having worked on a similar case in the previous probe, this probe was easy to complete.
+
+== Outcome
+
+The probe is successfully merged and live into production.
+
+== Conclusion
```

Comment:
This is a really simple probe and there isn't much to describe. The information in the "Conclusion" section is repetitive here. Would it be fine to remove it?